docs/configuration-options.md

```diff
@@ -0,0 +1,124 @@
+
+### Environment variables for Community Edition and Enterprise Server
+
+The following environment variables apply to **Mend Renovate Community Edition** and the **Mend Renovate Enterprise Edition Server**.<br/>
+Environment variables for the **Mend Renovate Enterprise Worker** are in the next section.
+
+#### Mend licensing config
+
+**`MEND_RNV_ACCEPT_TOS`**: Set this environment variable to `y` to consent to [Mend's Terms of Service](https://www.mend.io/terms-of-service/).
+
+**`MEND_RNV_LICENSE_KEY`**: Contact Mend to request a license key. [Mend.io/renovate](https://www.mend.io/renovate/)
+
+**`MEND_RNV_MC_TOKEN`**: [Enterprise only] The merge confidence token used for Smart-Merge-Control authentication.
+
+#### Source code management (SCM) connection details
+
+This section contains configuration variables for connecting to your source code repository.
+Use the appropriate settings to define connection details to your specific SCM.
+
+**`MEND_RNV_PLATFORM`**: The type of SCM. Options: github, gitlab, bitbucket.
+
+**`MEND_RNV_ENDPOINT`**: This is the API endpoint for your SCM. Required for self-hosted SCMs; not for GitHub.com. Include the trailing slash.
+
+**`MEND_RNV_GITHUB_APP_ID`**: [GitHub only] The GitHub App ID of the provisioned Renovate app on GitHub.
+
+**`MEND_RNV_GITHUB_APP_KEY`**: [GitHub only] A string representation of the private key of the provisioned Renovate app on GitHub. To insert the value directly into a Docker Compose environment variable, open the PEM file in a text editor and replace all new lines with "\n" so that the entire key is on one line. Alternatively, you can skip setting this key as an environment variable and instead mount it as a file to the path specified by `RNV_GITHUB_PEM_FILE_PATH`, as shown in the example Docker Compose file.
+
+**`RNV_GITHUB_PEM_FILE_PATH`**: [GitHub only] The file path for GitHub app key. Defaults to `/usr/src/app/renovate.private-key.pem`.
+
+**`MEND_RNV_GITLAB_PAT`**: [GitLab only] Personal Access Token for the GitLab bot account.
+
+**`MEND_RNV_WEBHOOK_SECRET`**: Optional: Defaults to `renovate`
+
+#### Optional Mend Renovate configuration
+
+**`GITHUB_COM_TOKEN`**: A Personal Access Token for a user account on github.com (i.e. _not_ an account on your GitHub Enterprise instance).
+This is used for retrieving changelogs and release notes from repositories hosted on github.com and it does not matter who it belongs to.
+It needs only read-only access privileges. Not required if SCM is GitHub.com.
+
+**`MEND_RNV_SERVER_API_SECRET`**: [Required if APIs enabled. Required on Renovate Enterprise Server] Set an API secret. Must match the Worker instances and Admin APIs for communicating with the Server.
+
+**`MEND_RNV_ADMIN_API_ENABLED`**: Optional: Set to 'true' to enable Admin APIs. Defaults to 'false'.
+
+**`MEND_RNV_SERVER_PORT`**: The port on which the server listens for webhooks and api requests. Defaults to 8080.
```

Comment:
Optional? EE only?

Reply:
It's not EE only. It's optional for both CE and EE.
API_SECRET is required on EE so that the Worker can talk to the Server with internal APIs. But enabling Admin APIs is a user choice on both CE and EE.